File: fizzbuzz-test.js
function fizzBuzzTest(firstNumber, lastNumber) {
    console.log("FizzBuzz with Js")
    for (let i = firstNumber; i <= lastNumber; i++) {
        if (i % 15 == 0) {
            console.log("FizzBuzz");
        } else if (i % 5 == 0) {
            console.log("Buzz");
        } else if (i % 3 == 0) {
            console.log("Fizz");
        } else {
            console.log(i);
        }
    }
}

fizzBuzzTest(1, 100);

File: fizzbuzz-test.go
package main

import (
	"fmt"
)

func fizzBuzzTest(firstNumber, lastNumber int) {
	fmt.Println("FizzBuzz with Golang")
	for i := firstNumber; i <= lastNumber; i++ {
		if i % 15 == 0 {
			fmt.Println("FizzBuzz")
		} else if i % 5 == 0 {
			fmt.Println("Buzz")
		} else if i % 3 == 0 {
			fmt.Println("Fizz")
		} else {
			fmt.Println(i)
		}
	}
}

func main() {
	fizzBuzzTest(1, 100)
}


